A32NX/layout.json

```diff
@@ -210,34 +210,39 @@
             "size": 313408,
             "date": "132402817714110148"
         },
+        {
+            "path": "html_ui/Fonts/wwDigital.ttf",
+            "size": 59612,
+            "date": "132402817714110148"
+        },
         {
             "path": "html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/BAT/A320_Neo_BAT.css",
-            "size": 1437,
+            "size": 1732,
             "date": "132402817714110148"
         },
         {
             "path": "html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/BAT/A320_Neo_BAT.html",
-            "size": 1280,
+            "size": 1361,
             "date": "132402817714110148"
         },
         {
```

Comment:
you don't have to commit this file anymore. CI takes care of updating it after merge. please remove the changes to this file from the PR.

Reply:
fixed